### DISCUSSION TITLE
Printer Request - Snapmaker J1

### DISCUSSION THREAD

**Discussion by walkinthepark24** [OrcaSlicer/OrcaSlicer#765] [General]
If possible, can the Snapmaker J1 be a printer that is added to Orca Slicer in the near future?

**Comment by PepeBumbulab** [comment]
Snapmaker J1 and Artisan support would be lovely in the future, the Orca Slicer is just so nice to use and way way more intuitive than luban or cura :)

**Comment by hliebscher** [comment]
Snapmaker J1 is added in config .... Artisan and SM2 dualHead works with a custom config
